pkg/daemon/ceph/osd/volume.go

```diff
@@ -142,6 +142,7 @@ func updateLVMConfig(context *clusterd.Context) error {
 	output := bytes.Replace(input, []byte("udev_sync = 1"), []byte("udev_sync = 0"), 1)
 	output = bytes.Replace(output, []byte("udev_rules = 1"), []byte("udev_rules = 0"), 1)
 	output = bytes.Replace(output, []byte("use_lvmetad = 1"), []byte("use_lvmetad = 0"), 1)
+	output = bytes.Replace(output, []byte("obtain_device_list_from_udev = 1"), []byte("obtain_device_list_from_udev = 0"), 1)
```

Comment:
Please remove this commit (it's already in master) and rebase your commit on master.

Reply:
Done, sorry for miss.